qiskit_nature/second_q/algorithms/excited_states_solvers/excited_states_solver.py

```diff
@@ -58,7 +58,7 @@ def get_qubit_operators(
     ) -> Tuple[PauliSumOp, Optional[dict[str, PauliSumOp]]]:
         """Construct qubit operators by getting the second quantized operators from the problem
         (potentially running a driver in doing so [can be computationally expensive])
-        and using a QubitConverter to map and reduce the operators to qubit operators.
+        and using a QubitConverter or QubitMapper to map and reduce the operators to qubit operators.
```

Comment:
We will need to do a thorough pass of the docs. Just mentioning this so its on our radar (I know that this is still a draft PR)